app/src/main/java/edu/uc/app/parkfinder/service/ParkService.kt

```diff
@@ -31,8 +32,7 @@ class ParkService {
              * Note: An HTTP response may still indicate an application-level failure such as a 404 or 500.
              * Call [Response.isSuccessful] to determine if the response indicates success.
              */
-            override fun onResponse(call: Call<ArrayList<Park>>, response: Response<ArrayList<Park>>
-            ) {
+            override fun onResponse(call: Call<ArrayList<Park>>, response: Response<ArrayList<Park>>) {
```

Comment:
Moved parenthesis to proper location at the end of method parameters and put open bracket next to it (Kotlin best practices)